admin-auth/src/test/kotlin/com/epam/drill/admin/auth/EnvRepositoryTest.kt

```diff
@@ -38,10 +36,13 @@ class EnvRepositoryTest {
     }
 
     @Test
-    fun `given users from env config, findAllNotDeleted must return all users not deleted users`() = runBlocking {
-        val repository = getRepositoryWithUsers(USER1, USER2)
+    fun `given users from env config, findAll must return all users not deleted users`() = runBlocking {
```

Comment:
rename to `given users from env config, findAll must return all users` - `EnvRepository` does not support deleting users